server/devtool/src/main/java/org/eclipse/vorto/server/devtool/service/IInformationModelEditorService.java

```diff
@@ -0,0 +1,21 @@
+package org.eclipse.vorto.server.devtool.service;
+
+import java.util.List;
+import java.util.Set;
+
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.vorto.http.model.ModelId;
+import org.eclipse.vorto.http.model.ModelResource;
+
+/**
+ * 
+ * @author shiv
+ *
+ */
+public interface IInformationModelEditorService {
+
+	String linkFunctionBlockToInformationModel(String infoModelResourceId, ModelId functionBlockModelId, ResourceSet resourceSet,
+			Set<String> referencedResourceSet);
+
+	List<ModelResource> searchFunctionBlockByExpression(String expression);
```

Comment:
Hi Shiv, 

if you require this due to the reasons you mentioned ,then I understand and you should leave it as such.
I am just trying to understand.

Thanks for the clarifications.